src/viadot/orchestration/prefect/flows/epicor_to_parquet.py

```diff
@@ -0,0 +1,64 @@
+"""Flows for downloading data from Epicor Prelude API to Parquet file."""
+
+from typing import Literal
+
+from prefect import flow
+
+from viadot.orchestration.prefect.tasks import epicor_to_df
+from viadot.orchestration.prefect.tasks.task_utils import df_to_parquet
+
+
+@flow(
+    name="extract--epicor--parquet",
+    description="Extract data from Epicor Prelude API and load it into Parquet file",
+    retries=1,
+    retry_delay_seconds=60,
+)
+def epicor_to_parquet(
+    path: str,
+    base_url: str,
+    filters_xml: str,
+    if_exists: Literal["append", "replace", "skip"] = "replace",
+    validate_date_filter: bool = True,
+    start_date_field: str = "BegInvoiceDate",
+    end_date_field: str = "EndInvoiceDate",
+    epicor_credentials_secret: str | None = None,
+    epicor_config_key: str | None = None,
+) -> None:
+    """Download a pandas `DataFrame` from Epicor Prelude API load it into Parquet file.
+
+    Args:
+        path (str): Path to Parquet file, where the data will be located.
+            Defaults to None.
+        base_url (str, required): Base url to Epicor.
+        filters_xml (str, required): Filters in form of XML. The date filter
+            is required.
```

Comment:
An example usage below would be great (Examples section in docsting, sth like this https://github.com/dyvenia/viadot/blob/2.0/src/viadot/sources/trino.py#L86-L93) or a link to docs describing how this param should look like.